src/frontend/pages/game.tsx

```diff
@@ -15,6 +15,8 @@ import Colors from '../constants/colorConstants';
 import { API, API_URL } from '../api-client';
 import { useRouter } from 'next/dist/client/router';
 import { useEventSource } from '../hooks/useEventSource';
+import toast from 'toasted-notes';
+import 'toasted-notes/src/styles.css';
```

Comment:
I'm curious to know if we actually need to do this. I'm assuming these are default styles. Why do they additionally need to be imported?

Reply:
You'd think you wouldn't, but the npm package requires importing the styles too 🤷